proto/nibiru/perp/v2/tx.proto

```diff
@@ -429,3 +434,19 @@ message MsgShiftSwapInvariant {
 }
 
 message MsgShiftSwapInvariantResponse {}
+
+// -------------------------- WithdrawFromPerpFund --------------------------
+
+// WithdrawFromPerpFund: gRPC tx msg for changing the swap invariant.
+// Admin-only.
+message MsgWithdrawFromPerpFund {
+  string sender = 1;
+  string amount = 2 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  string denom = 3;
+  string to_addr = 4;
```

Comment:
(nit) it might be safer to just withdraw to whoever sends the message, i.e. the message `sender`.

I can foresee possible human error in the future where somebody sends the message but accidentally sets the `to_addr` different from the message signer/sender.

Reply:
I think so too. The problem we might encounter here is that the claim goes to a smart contract that doesn't expose a function to withdraw its balance. 

`sudo_contract_calls_msg -> withdraw_to_contract_addr -> No fn to withdraw -> funds are lost`

I'm trying to avoid this ^ deadlock.

Adding an explicit `to_addr` makes it less error prone in a different way, by making the caller pass in the location they want the funds to end up. 

Another option is to make that field nullable, so that empty means, `to_addr := sender`. Would you prefer that?

Reply:
Wow, I didn't even consider that scenario. In that case, then I think it's better to be explicit. 

Thinking about it more, in the general case, we'll likely be sending these `WithdrawFromPerpFund` messages from the CW3 multisig. Having the ability to target a recipient with the message would be more efficient, instead of withdrawing to the CW3 multisig and then doing a bank transfer.

A more sophisticated design could be to only run the message if the `to_addr` is part of the `sudo` sudoers list. It would prevent the funds from going to a CosmWasm smart contract that doesn't have the withdraw functionality, because we would be the ones whitelisting the possible recipients. But that could be addressed in a separate PR. What's implemented looks good to me.